src/Microsoft.AspNetCore.ResponseCaching/KeyProvider.cs

```diff
@@ -153,5 +151,28 @@ public virtual string CreateVaryKey(HttpContext httpContext, VaryRules varyRules
                 _builderPool.Return(builder);
             }
         }
+
+        public virtual string CreateBodyKey(HttpContext httpContext)
```

Comment:
I think you can just remove this method, we don't expect it to exist in the next itteration